js/interface.js

```diff
@@ -1720,6 +1729,77 @@ $('#show-versions').click(function() {
   fetchCurrentDataSourceVersions();
 });
 
+function findSecurityRule() {
+  var rule = currentDataSourceRules.map(rule => {
+    if (rule.allow.hasOwnProperty('tokens') && rule.allow.tokens.indexOf(widgetData.tokenId) !== -1) {
+      return rule;
+    }
+  });
+
+  return rule;
+}
+
+function getSelectedTokenDetails() {
+  var tokenSelectedName;
+  var tokenSelectedId;
+
+  var tokenDetails = _.find(integrationTokenList, function(itl) {
```

Comment:
`itl`?

Reply:
did the changes as you suggested. actually I used the initial letter of variable name as function parameter. We will take care this onwards.